frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -421,409 +423,377 @@ class BitBox02 extends Component<Props, State> {
-                                                </div>
-                                                <div className="m-top-quarter">
-                                                    <Checkbox
-                                                        onChange={this.handleDisclaimerCheck}
-                                                        className={style.wizardCheckbox}
-                                                        id="agreement3"
-                                                        checked={agreement3}
-                                                        label={t('bitbox02Wizard.backup.userConfirmation3')} />
-                                                </div>
-                                                <div className="m-top-quarter">
-                                                    <Checkbox
-                                                        onChange={this.handleDisclaimerCheck}
-                                                        className={style.wizardCheckbox}
-                                                        id="agreement4"
-                                                        checked={agreement4}
-                                                        label={t('bitbox02Wizard.backup.userConfirmation4')}/>
-                                                </div>
-                                                <div className="m-top-quarter">
-                                                    <Checkbox
-                                                        onChange={this.handleDisclaimerCheck}
-                                                        className={style.wizardCheckbox}
-                                                        id="agreement5"
-                                                        checked={agreement5}
-                                                        label={t('bitbox02Wizard.backup.userConfirmation5')}/>
-                                                </div>
-                                            </form>
-                                            <div className={['buttons text-center', style.fullWidth].join(' ')}>
-                                                <Button
-                                                    primary
-                                                    onClick={this.createBackup}
-                                                    disabled={creatingBackup || !readDisclaimers}>
-                                                    {t('securityInformation.create.button')}
-                                                </Button>
-                                            </div>
-                                        </div>
-                                        <div className="text-center m-top-large">
-                                            <SwissMadeOpenSource large />
-                                        </div>
-                                    </Step>
+            <Main>
```

Comment:
Found a bug when canceling set-name on the bitbox it currently doens't show the aborted by the user dialog, I believe View component covers it.

Reply:
Fixed in https://github.com/digitalbitbox/bitbox-wallet-app/pull/1601/commits/03bcc2e4c02fb692d4d366e50a42569359a2cbd7